pkg/api/handlers/compat/images.go

```diff
@@ -503,7 +503,7 @@ func LoadImages(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(loadReport.Names) != 1 {
+	if len(loadReport.Names) < 1 {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Errorf("%d instead of 1 were loaded", len(loadReport.Names)))
```

Comment:
This error message makes no sense now.

Reply:
@rhatdan Ackd. Changed it so it makes more sense in this scenario.